srcpkgs/libaccounts-qt5/template

```diff
@@ -13,7 +13,10 @@ license="LGPL-2.1-only"
 homepage="https://gitlab.com/accounts-sso/libaccounts-qt"
 distfiles="${homepage}/repository/archive.tar.gz?ref=VERSION_${version}>${pkgname}-${version}.tar.gz"
 checksum=cfa826076e04a7d075baac12e73f4e890a3401ac30c7f80ed9e257a5c6bfbfef
-nocross="libaccounts-glib can't be crosscompiled"
+
+if [ "$CROSS_BUILD"  ]; then
+ hostmakedepends+=" qt5-qmake qt5-host-tools"
```

Comment:
Just add them unconditionally. They are strictly speaking always needed. Just implicit right now in makedepends when not cross compiling.